impala/thrift/process_thrift.sh

```diff
@@ -39,7 +45,7 @@ grep -v 'namespace py beeswaxd' $IMPALA_REPO/common/thrift/beeswax.thrift \
 
 # hive_metastore.thrift assumes a directory structure for fb303.thrift, so we
 # change the include statement here
-cat $IMPALA_REPO/thirdparty/hive-*/src/metastore/if/hive_metastore.thrift \
+cat $IMPALA_TOOLCHAIN/cdh_components/hive-$IMPALA_HIVE_VERSION/src/metastore/if/hive_metastore.thrift \
```

Comment:
This doesn't work on a clean environment - cdh_components now frequently has a version after it.


$HIVE_SRC_DIR/metastore/if/hive_metastore.thrift should work

Reply:
After making that change there was some additional generated thrift code for the GetPartitions() method